docs/admin/templates/extending-templates/prebuilt-workspaces.md

```diff
@@ -378,11 +378,17 @@ This keeps other provisioners available to handle user-initiated jobs.
 #### Setup
 
 1) Create a provisioner key with a prebuild tag (e.g., `is_prebuild=true`).
+
 Provisioner keys are org-scoped and their tags are inferred automatically by provisioner daemons that use the key.
+
+**Note:** `coder_workspace_tags` are cumulative, so if your template already defines provisioner tags, you will need to create the provisioner key with the same tags plus the `is_prebuild=true` tag so that prebuild jobs correctly match the dedicated prebuild pool.
```

Comment:
This might also be good to add into `docs/admin/templates/extending-templates/workspace-tags.md`